imaging/ml/ml_codelab/breast_density_auto_ml.ipynb

```diff
@@ -163,9 +147,9 @@
     "id": "HKsCfbXorosM"
    },
    "source": [
-    "Next, we are going to transfer the DICOM instances from the TCIA API to the Cloud Healthcare API. We have added a helper script called *[store_tcia_in_hc_api.py](./scripts/store_tcia_in_hc_api.py)* to do this. Internally, this uses the STOW-RS DICOMWeb protocol (implemented as DicomWebPost in Cloud Healthcare API).\n",
```

Comment:
Ditto: Lets do this in a separate PR. Also, when we do switch to the hosted dataset, I don't think there will be any need to transfer - the dataset is already hosted in CHC API (as well as in GCS).

Reply:
https://github.com/GoogleCloudPlatform/healthcare/issues/433